cos/static/modal.css

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
I hate to do this to you with all the git problems you have had, but I don't think this is being used (bootstrap is taking care of this, and it is commented out in game.jinja2) and can be removed.

Reply:
Actually, it looks like this is still being used in landingpage.jinja2. We should probably at some point pull that out and replace it with the corrected bootstrap as was done in game.jinja2, but that can probably be done in a later ticket. What are your thoughts?

Reply:
Yea, I have several custom functions in the modal.css (including modal size and dismiss on click outside) that would need to be ported. I think this is good for later review.